pkg/rpc/client/rpc.go

```diff
@@ -499,16 +500,19 @@ func (c *Client) AddNetworkFee(tx *transaction.Transaction, extraFee int64, accs
 	}
 	size := io.GetVarSize(tx)
 	for i, cosigner := range tx.Signers {
-		if accs[i].Contract.Script == nil {
-			contract, err := c.GetContractState(cosigner.Account)
-			if err == nil {
-				if contract == nil {
-					continue
+		if accs[i].Contract.Deployed {
```

Comment:
So how do I create a wallet for deployed contract that I could use with standard CLI commands?

Reply:
Fixed.